### PR TITLE
update dataflow python container

### PR DESCRIPTION
**Pull request by aaltay**
R: @chamikaramj

### CODE REVIEW

File: sdks/python/apache_beam/runners/dataflow/internal/dependency.py [hunk 1]
```diff
@@ -72,7 +72,8 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to the execution environment.
-BEAM_CONTAINER_VERSION = '2.0.0'
+# This should be in the beam-[version]-[date] format, date is optional.
+BEAM_CONTAINER_VERSION = 'beam-2.1.0-20170518'
```
Comment (chamikaramj) [inline sdks/python/apache_beam/runners/dataflow/internal/dependency.py]: Should this be beam-2.0.1-date (in case we have to cut such a release) ?

Comment (aaltay) [inline sdks/python/apache_beam/runners/dataflow/internal/dependency.py]: We are already at version 2.1.0.dev (https://github.com/apache/beam/blob/master/sdks/python/apache_beam/version.py#L21) I think this should match the current head version.

**Comment by chamikaramj** [comment]
LGTM

**Comment by aaltay** [comment]
Run Python PostCommit

**Pull request closed without merging**